utilities/write_batch_with_index/write_batch_with_index.cc

```diff
@@ -328,16 +328,19 @@ Iterator* WriteBatchWithIndex::NewIteratorWithBase(
                                   &rep->comparator);
   }
 
-  return new BaseDeltaIterator(column_family, base_iterator, wbwiii,
-                               GetColumnFamilyUserComparator(column_family));
+  return new BaseDeltaIterator(
```

Comment:
nit: shall we just make `BaseDeltaIterator` just take the `read_options`, in case there are future options that should be aligned between base and this iterator.

Reply:
You mean just changing the constructor signature, right?

Reply:
Yes, changing it to take the whole read options pointer, as opposed to this specific option.

Reply:
Yeah, that's a good idea, thanks!